packages/node/src/utils/schema.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { camelCase } from 'lodash';
+
+export interface indexField {
+  schemaName: string;
+  tableName: string;
+  entityName: string;
+  fieldName: string;
+  isUnique: boolean;
+  type: string;
+}
+
+export async function packEntityFields(
+  schema: string,
+  entity: string,
+): Promise<indexField[]> {
+  const model = this.sequelize.model(entity);
+
+  const rows = await this.sequelize.query(
+    `select
+    n.nspname as schema_name ,
+    tab.relname as table_name,
+    '${entity}' as entity_name,
```

Comment:
Need a test, I think entity name is converted into underscore case when creating table